src/app/pages/dashboard/components/dashboard/dashboard.component.ts

```diff
@@ -75,7 +75,7 @@ export class DashboardComponent implements OnInit {
   protected onAddGroup(): void {
     const newGroup: WidgetGroup = {
       layout: WidgetGroupLayout.Full,
-      slots: [],
+      slots: [{ category: null, type: null }],
```

Comment:
Technically, we should be adding a new group here at all. But we're doing that and we are adding the new group to the dashboard right here. But since, we're doing that, a full layout widget has at least 1 slot and the object should represent that. I will remove the whole logic from here.